cmake/CCCLInstallRules.cmake

```
@@ -0,0 +1,10 @@
+# Bring in CMAKE_INSTALL_LIBDIR
+include(GNUInstallDirs)
+
+# CCCL has no installable binaries, no need to build before installing:
+set(CMAKE_SKIP_INSTALL_ALL_DEPENDENCY TRUE)
+
+install(DIRECTORY "${CCCL_SOURCE_DIR}/share/cmake/cccl"
```

```suggestion
install(DIRECTORY "${CCCL_SOURCE_DIR}/lib/cmake/cccl"
```
And move the files to `lib` directory so we have the same source and install layout